package.json

```diff
@@ -10,6 +10,7 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
+    "setup:project": "node src/scripts/setup-env.files.js",
```

Comment:
not sure about the script name here.

Reply:
maybe we need to change this script name + the script file name because they are coupled to the onboarding flow.